functions/global-setup.ts

```diff
@@ -4,7 +4,7 @@ module.exports = async function globalSetup() {
   globalThis.servers = await setup({
     command: `yarn start:cloud`,
     port: 3000,
-    launchTimeout: 80000,
+    launchTimeout: 120000,
```

Comment:
Add a comment justifying this:

```suggestion
    launchTimeout: 120000, // takes ~2m on CI
```